packages/reporting-e2e/tests/2-reporting.test.ts

```diff
@@ -0,0 +1,99 @@
+import { getReport, login, overwriteClick } from './utils';
+import { rmfData } from './mock-data';
+import fetch from 'node-fetch';
+
+describe('reporting interactions', () => {
+  before(() => overwriteClick());
+  before(() => browser.url('/'));
+
+  before(login);
+
+  it('should populate the database', async () => {
+    const options = {
+      method: 'POST',
+      body: JSON.stringify(rmfData),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+
+    let response;
+    try {
+      const res = await fetch(`http://localhost:8003/log/rmfserver`, options);
+      response = await res.text();
+    } catch (error) {
+      console.log(error);
+    }
+
+    expect(response).toBe('"Logs were saved correctly"');
+  });
```

Comment:
I think this should be called: `stores logs correctly`. 

and populate the database as well in a `beforeEach`. Because we are depending on this test to be executed first for the other to pass. Tests should be executed in different orders and it should work.

or you can use this here `2-reporting.test.ts`: 

```
before(async () => {
    const options = {
      method: 'POST',
      body: JSON.stringify(rmfData),
      headers: {
        'Content-Type': 'application/json',
      },
    };

    try {
      const res = await fetch(`http://localhost:8003/log/rmfserver`, options);
      return await res.json();
    } catch (error) {
      console.log(error);
    }
  });
```

And create another test file `3-logs-creation.test.ts`  to test `it('should populate the database', async () => {`